app/views/signatories/new.html.erb

```diff
@@ -1 +1,8 @@
-<h1>signatories</h1>
\ No newline at end of file
+<h1>Add a signatory to a contract</h1>
+
+<%= simple_form_for @individual do |f| %>
+  <%= f.input :individual, collection: Individual.all, input_html: { class: "select2", style: "width: 100%"}, label_html: { class: "row justify-content-center"} %>
```

Comment:
I 'll be able to complet this part only whenever will be able to create contracts